setup.cfg

```diff
@@ -71,9 +71,13 @@ exclude =
 # https://setuptools.readthedocs.io/en/latest/setuptools.html#options
 # (see notes for the asterisk/`*` meaning)
 * =
-#    *.c
     *.so
 
+[options.exclude_package_data]
```

Comment:
plz move this before `extra_requires` to keep the sections sorted alphabetically.